draft-00.xml

```diff
@@ -488,22 +488,16 @@
             <xref target="prop-href">'href'</xref>
             and
             <xref target="prop-format">'format'</xref>.
-            If the 'href' property appears it SHOULD contain
-            a dereferencable URL that points to human-readable
-            text. If the 'format' property appears it SHOULD
-            contain one of the following values:
-            <list style="numbers">
-              <t>"text"</t>
-              <t>"html"</t>
-              <t>"asciidoc"</t>
-            </list>
+            If the 'href' property appears it SHOULD contain a
+            dereferencable URL that points to human-readable text. If
+            the 'format' property appears it SHOULD contain one of the
+            following values: "text", "html", "asciidoc", or "markdown".
```

Comment:
@fosrias : yeah - single/double usage is inconsistent throughout. we can do another issue/PR to clear it up doc-wide.